style.css

```diff
@@ -0,0 +1,1014 @@
+}
+
+.menu-btn.open .menu-btn__burger::after {
+  transform: translate(20px, 0) rotate(-45deg);
+}
+
+/* ============== invitation section ============ */
+.invitation {
+  background-image: url("./img/invitationbackg-1.png");
+  background-size: cover;
+  background-position: center;
+  overflow: hidden;
+}
+
+.overlay {
+  background-color: var(--background-color);
+}
+
+.invitation-bg {
+  background-image: url("./img/invitationbackg-1.png");
+  background-position: center;
+  background-size: cover;
+  padding: 5rem 0;
+}
+
+.invitation-content .welcome {
+  color: var(--theme);
+  font-weight: 400;
+  margin-bottom: 0.5rem;
+}
+
+.invitation-content h1 {
+  color: var(--theme);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-image: url(./img/font-color.jpg);
+  font-weight: 900;
+  font-size: 50px;
+  line-height: 1;
+  margin-bottom: 1rem;
+}
+
+.invitation-content p {
+  color: var(--navy);
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.5;
+  margin-bottom: 0.5rem;
+  padding: 0.5rem;
+  background-color: var(--white-color);
+  border: 2px solid var(--white-color);
+  max-width: 42rem;
+  margin-top: 3rem;
+}
+
+.invitation-content .realTime {
+  color: var(--ash);
+  font-weight: 900;
+  font-size: 30px;
+}
+
+.invitation-content .venue {
+  font-size: 14px;
+  font-weight: 500;
+}
+
+/* ======== Program css =========== */
+.activity {
+  background-color: var(--navy);
+  background-image: url(./img/cross.svg);
+  background-size: 32px;
+  color: var(--white-color);
+  padding: 1rem 0;
+  overflow: hidden;
+}
+
+.activity-content {
+  padding: 2rem 0;
+  position: relative;
+}
+
+.mainTitle {
+  text-align: center;
+  margin-bottom: 2rem;
+  font-weight: 500;
+  position: relative;
+  padding: 1rem;
+  padding-bottom: 1.5rem;
+}
+
+.mainTitle::before {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  width: 50px;
+  right: 45%;
+  height: 1px;
+  background-color: var(--theme);
+  margin-bottom: 1rem;
+}
+
+.activity-list {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  gap: 1rem;
+}
+
+.activity-item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  cursor: pointer;
+  padding: 0.5rem 1rem;
+  background-color: var(--background);
+  transition: all 0.3s ease-in-out;
+}
+
+.activity-item:hover {
+  border: 1px solid var(--white-color);
+  transform: scale(1.02);
+}
+
+.activity-item .activity-item-img {
+  width: auto;
+  height: auto;
+  overflow: hidden;
+}
+
+.activity-img {
+  max-width: 100px;
+}
+
+.activity-item .activity-item-text {
+  padding: 0 0.5rem;
+  width: 50%;
+}
+
+.activity-item h3 {
+  font-size: 16px;
+  color: var(--theme);
+  display: flex;
+  margin: auto;
+}
+
+.guest-item-text p {
+  font-size: 12px;
+  font-weight: 700;
+  max-width: 20rem;
+}
+
+.activity-item .activity-item-text p {
+  font-size: 14px;
+  display: flex;
+  margin: auto;
+}
```

Comment:
- [ ] [OPTIONAL] Great job 👍 making your images 🖼️ responsive as was requested by the previous reviewer. It would be better if you could also make your text content align properly. Please also make your `font-sizes `and `font weights` look like [design guideline](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) .Non-responsive design, and inconsistent typefaces are a few hallmarks of bad website design.

Behance desgn:

<img width="696" alt="Screenshot 2023-03-14 at 12 58 13 PM" src="https://user-images.githubusercontent.com/44798044/224936361-92210a26-8fbd-4c71-a183-478b624d0169.png">

<img width="364" alt="Screenshot 2023-03-14 at 12 57 56 PM" src="https://user-images.githubusercontent.com/44798044/224937617-57588705-dfcb-4f72-9226-8e15c159fdf6.png">


Your design:

<img width="1441" alt="Screenshot 2023-03-14 at 12 07 05 PM" src="https://user-images.githubusercontent.com/44798044/224936669-60a139a1-6f32-49bc-bb61-f50890806026.png">

<img width="338" alt="Screenshot 2023-03-14 at 12 06 20 PM" src="https://user-images.githubusercontent.com/44798044/224937652-ffca5ef1-38b6-41a9-9781-c9a638252682.png">